src/modules/utils.js

```diff
@@ -0,0 +1,19 @@
+const toggleDisplay = (elemGroup1, elemGroup2) => {
+  elemGroup1.forEach((elem) => {
+    elem.classList.toggle('d-none');
+  });
+  elemGroup2.forEach((elem) => {
+    elem.classList.toggle('d-none');
+  });
+};
+
+const createTask = (task, index) => {
+  const id = index;
+  return {
+    description: task,
+    completed: false,
+    id,
+  };
+};
```

Comment:
Great work on your project 👍 
In this piece of code you can make the code cleaner by substituting the index parameter for the id and simplifying the arrow function by not using the return keyword.